.github/workflows/publish-npm-nightly.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish to NPM registry Nightly
+
+on:
+  schedule:
+    # GMT+8 00:00
+    - cron: '0 16 * * *'
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Add dev branch
+        run: git branch dev origin/dev
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: latest
+      - name: Install dependencies
+        run: pnpm i --frozen-lockfile
+      - name: Lint
+        run: pnpm lint
+      - name: Test
+        run: pnpm test
+
+  publish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          registry-url: https://registry.npmjs.com/
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: latest
+
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y%m%d')"
+
+      - name: Get git head
+        run: echo "GIT_HEAD=${GITHUB_SHA}" >> $GITHUB_ENV
+      - name: Gen npmrc
+        run: echo "//registry.npmjs.com/:_authToken=${{ secrets.NPM_PUBLISH_TOKEN }}" >> ./.npmrc
+
+      - name: Set Nightly
+        run: sh ./scripts/nightly.sh
+
+      - name: Build & publish
+        run: sh ./scripts/publish.sh
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_PUBLISH_TOKEN}}
+          TAG_VERSION: 1.0.${{ steps.date.outputs.date }}
```

Comment:
Can we do the format like `1.1.0-yyyy-mm-dd` ?

Reply:
According to semver, it should be `x.x.x` (x is number). Other packages like `caniuse-lite` is the same.

Reply:
Yeah I mean we can do something like the one that we are doing right now:

`1.1.0-beta.ver` 

the nightly version can go with:

`1.1.0-nightly.yyyymmdd`

Reply:
I don't think `nightly` version should add `1.1.0`. It means nothing.